scripts/drone/steps/lib.star

```diff
@@ -9,14 +9,14 @@ load(
 )
 
 grabpl_version = "v3.0.21"
-build_image = "grafana/build-container:v1.7.1"
+build_image = "grafana/build-container:1.7.2"
```

Comment:
did we changed how we tag build-container image without v?

Reply:
only 1.7.0 and 1.7.1 are prefixed with v, and I’m not quite sure why that was. In general, we don’t prefix Docker version tags afaict.

https://hub.docker.com/r/grafana/build-container/tags